### ISSUE TITLE
Can't load the package in Atom 1.10

### ISSUE THREAD

**Issue by alexsanderdesouza** [yongkangchen/atom-ctags#172]
**Atom Version**: 1.10.0-beta0
**Electron Version**: 0.37.8
**System**: Debian GNU/Linux
**Thrown From**: [atom-ctags](https://github.com/yongkangchen/atom-ctags) package, v4.5.2

### Stack Trace

Failed to activate the atom-ctags package

```
At Cannot find module '/usr/share/atom-beta/resources/app.asar.unpacked/node_modules/symbols-view/node_modules/ctags/build/Release/ctags.node'

Error: Cannot find module '/usr/share/atom-beta/resources/app.asar.unpacked/node_modules/symbols-view/node_modules/ctags/build/Release/ctags.node'
    at Module._resolveFilename (module.js:339:15)
    at Function.Module._resolveFilename (/usr/share/atom-beta/resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (/usr/share/atom-beta/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/home/alexsander/.atom/packages/atom-ctags/node_modules/ctags/src/ctags.coffee:1:10)
    at Object.<anonymous> (/home/alexsander/.atom/packages/atom-ctags/node_modules/ctags/src/ctags.coffee:1:1)
    at Module._compile (/usr/share/atom-beta/resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (/usr/share/atom-beta/resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (/usr/share/atom-beta/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/home/alexsander/.atom/packages/atom-ctags/lib/ctags-cache.coffee:3:9)
    at Object.<anonymous> (/home/alexsander/.atom/packages/atom-ctags/lib/ctags-cache.coffee:2:1)
    at Module._compile (/usr/share/atom-beta/resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (/usr/share/atom-beta/resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (/usr/share/atom-beta/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.module.exports.activate (/home/alexsander/.atom/packages/atom-ctags/lib/main.coffee:48:19)
    at Package.module.exports.Package.activateNow (/usr/share/atom-beta/resources/app.asar/src/package.js:183:20)
    at /usr/share/atom-beta/resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/usr/share/atom-beta/resources/app.asar/src/package.js:92:15)
    at /usr/share/atom-beta/resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/usr/share/atom-beta/resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/usr/share/atom-beta/resources/app.asar/src/package-manager.js:538:21)
    at /usr/share/atom-beta/resources/app.asar/src/package-manager.js:519:29
    at Config.module.exports.Config.transactAsync (/usr/share/atom-beta/resources/app.asar/src/config.js:337:18)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "audioBeep": false,
    "autoHideMenuBar": true,
    "disabledPackages": [
      "package-generator",
      "open-on-github",
      "symbols-view",
      "deprecation-cop",
      "build-tools-cpp",
      "git-tab-status",
      "language-clojure",
      "language-csharp",
      "language-go",
      "language-mustache",
      "language-sass",
      "language-toml",
      "metrics",
      "autocomplete-css",
      "autocomplete-html",
      "wrap-guide",
      "activate-power-mode",
      "svn"
    ],
    "ignoredNames": [
      ".libs",
      ".git",
      ".svn",
      ".deps",
      "*.o",
      ".tags*",
      ".dirstamp"
    ],
    "projectHome": "/home/alexsander/workspace/buildroot/",
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ]
  },
  "atom-ctags": {
    "buildTimeout": 180000,
    "cmdArgs": "--exclude=m4 --exclude=docs --exclude=Makefile* --exclude=doc"
  }
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.29.10 (inactive)
atom-ctags, v4.5.2 (inactive)
atom-material-syntax, v0.4.6 (inactive)
atom-material-ui, v1.3.3 (inactive)
auto-copyright, v0.5.0 (inactive)
column-select, v0.2.0 (inactive)
file-icon-supplement, v0.8.3 (inactive)
fonts, v1.6.0 (inactive)
highlight-line, v0.11.1 (inactive)
highlight-selected, v0.11.2 (inactive)
indentation-indicator, v1.0.0 (inactive)
language-latex, v0.6.1 (inactive)
language-patch, v1.2.0 (inactive)
latex-autocomplete, v0.5.3 (inactive)
latextools, v0.8.1 (inactive)
minimap, v4.24.7 (inactive)
minimap-bookmarks, v0.3.1 (inactive)
minimap-git-diff, v4.3.1 (inactive)
multi-wrap-guide, v0.25.1 (inactive)
project-manager, v2.9.7 (inactive)
save-session, v0.15.7 (inactive)
svn, v0.0.12 (inactive)
switch-header-source, v0.20.0 (inactive)
symbols-tree-view, v0.13.2 (inactive)
todo-show, v1.7.0 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.5.0 (inactive)
one-light-ui, v1.5.0 (inactive)
one-dark-syntax, v1.3.0 (inactive)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.3 (inactive)
archive-view, v0.61.1 (inactive)
autocomplete-atom-api, v0.10.0 (inactive)
autocomplete-css, v0.11.2 (inactive)
autocomplete-html, v0.7.2 (inactive)
autocomplete-plus, v2.31.1 (inactive)
autocomplete-snippets, v1.11.0 (inactive)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (inactive)
background-tips, v0.26.1 (inactive)
bookmarks, v0.41.0 (inactive)
bracket-matcher, v0.82.1 (inactive)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (inactive)
dev-live-reload, v0.47.0 (inactive)
encoding-selector, v0.22.0 (inactive)
exception-reporting, v0.38.1 (inactive)
find-and-replace, v0.201.0 (inactive)
fuzzy-finder, v1.3.0 (inactive)
git-diff, v1.1.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (inactive)
image-view, v0.58.2 (inactive)
incompatible-packages, v0.26.1 (inactive)
keybinding-resolver, v0.35.0 (inactive)
line-ending-selector, v0.5.0 (inactive)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (inactive)
metrics, v0.53.1 (inactive)
notifications, v0.65.0 (inactive)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.241.0 (inactive)
snippets, v1.0.2 (inactive)
spell-check, v0.67.1 (inactive)
status-bar, v1.4.0 (inactive)
styleguide, v0.47.0 (inactive)
symbols-view, v0.113.0 (inactive)
tabs, v0.100.0 (inactive)
timecop, v0.33.2 (inactive)
tree-view, v0.208.2 (inactive)
update-package-dependencies, v0.10.0 (inactive)
welcome, v0.34.0 (inactive)
whitespace, v0.33.0 (inactive)
wrap-guide, v0.38.1 (inactive)
language-c, v0.52.1 (inactive)
language-clojure, v0.21.0 (inactive)
language-coffee-script, v0.47.2 (inactive)
language-csharp, v0.12.1 (inactive)
language-css, v0.37.1 (inactive)
language-gfm, v0.88.0 (inactive)
language-git, v0.15.0 (inactive)
language-go, v0.42.1 (inactive)
language-html, v0.45.1 (inactive)
language-hyperlink, v0.16.0 (inactive)
language-java, v0.23.0 (inactive)
language-javascript, v0.119.0 (inactive)
language-json, v0.18.2 (inactive)
language-less, v0.29.5 (inactive)
language-make, v0.22.2 (inactive)
language-mustache, v0.13.0 (inactive)
language-objective-c, v0.15.1 (inactive)
language-perl, v0.35.0 (inactive)
language-php, v0.37.2 (inactive)
language-property-list, v0.8.0 (inactive)
language-python, v0.45.0 (inactive)
language-ruby, v0.69.0 (inactive)
language-ruby-on-rails, v0.25.0 (inactive)
language-sass, v0.56.0 (inactive)
language-shellscript, v0.22.4 (inactive)
language-source, v0.9.0 (inactive)
language-sql, v0.23.0 (inactive)
language-text, v0.7.1 (inactive)
language-todo, v0.28.0 (inactive)
language-toml, v0.18.0 (inactive)
language-xml, v0.34.9 (inactive)
language-yaml, v0.26.0 (inactive)

# Dev
No dev packages
```

**Comment by zzjin** [comment]
same issue here. With mac os 10.11.5.

After search find that file `atom-beta/resources/app.asar.unpacked/node_modules/symbols-view/node_modules/ctags/build/Release/ctags.node` has been moved to `atom-beta/resources/app.asar.unpacked/node_modules/ctags/build/Release/ctags.node`

**Comment by zhaocai** [comment]
Atom 1.10.0 is live today, but they have not updated the `node-ctags` module to this the change.

For now, you can modify `node_modules\ctags\src\ctags.coffee` under `atom-ctags`.
Change the first line to

 `{Tags} = require(process.resourcesPath + '/app.asar.unpacked/node_modules/ctags/build/Release/ctags.node')`

**Comment by yongkangchen** [comment]
Fixed by c548440. Please update this package to version 5.0

**Comment by JustinAiken** [comment]
5.0 does indeed fix it, thanks! :)
